environment.yml

```diff
@@ -12,7 +12,7 @@ dependencies:
   - scikit-learn>=1.0
   - astropy>=4.3
   - setuptools>=46.4
-  - matplotlib>=3.4
+  - matplotlib>=3.7
```

Comment:
Did you bump the numpy version, as well?

Reply:
Yes, just did